exberry_adapter/bot/exberry_adapter_bot.py

```diff
@@ -52,6 +53,11 @@ def main():
     def say_hello(event):
         logging.info("DA Marketplace <> Exberry adapter is ready!")
 
+    @client.ledger_created(MARKETPLACE.ExberrySID)
+    def handleExberrySID(event):
+        global SID
```

Comment:
- What happens if you don't initialize it? it defaults to 100?
- Should the bot archive the contract after reading the info?
- Does the contract need a key to make it unique?

Please remove the # TODO at the top of the file as after this it will be done

Reply:
Yes - it defaults to `100`, and then after the contract is created, it changes the `SID` to whatever the contract was. If you create another contract while it's running, it will switch to that `SID`. In the README, it does point out that you have to create the contract _after_ starting the adapter.

I think it could make sense to archive the contract - after you create it and change the "starting" `SID`, that number will be out of date as soon as you perform a single order anyway.

I don't think there's a need for the contract key, especially if it gets archived. There's no need to ever lookup the contract, and since creating the contract is more of an action ("change the current starting SID to 150, please"), it doesn't matter if the contract is unique (again assuming the contract isn't archived).